tests/e2e/device-plugin_test.go

```diff
@@ -168,25 +148,23 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				targetNodeName := nodes[0].Name
-				Expect(targetNodeName).ToNot(BeEmpty(), "No suitable worker node found")
+				Expect(targetNodeName).ToNot(BeEmpty())
```

Comment:
I would say that this is probably useful additional context, why was it removed?